src/ops/Makefile

```diff
@@ -0,0 +1,58 @@
+# make sure RS_ENV is set and not 'dev'
+env_check:
+ifndef RS_ENV
+	@echo error: set RS_ENV to 'staging' or 'prod' 1>&2
+	@exit 1
+endif
+ifeq ($(RS_ENV),dev)
+	@echo error: RS_ENV can not be set to 'dev' 1>&2
+	@exit 1
+endif
+
+# remove source from docker image context tree
+clean:
+	rm -rf docker/expungeservice/backend \
+         docker/expungeservice/frontend
+
+# synchronize source trees into docker context tree
+rsync:
+	rsync -rlptoD --delete --omit-dir-times --exclude logs --exclude flask_session ../backend docker/expungeservice
+	rsync -rlptoD --delete --omit-dir-times ../frontend/build docker/expungeservice/frontend
+
+# echo out version.json
+version:
+	echo {\"recordexpungPDX\":\"$$(git rev-parse HEAD)\"} > docker/expungeservice/frontend/build/version.json
```

Comment:
Thanks for adding this

Reply:
little breadcrumbs like this are super helpful when the need arises. we can add more metadata to this json down the line as well.